app/views/_macros/components.nunjucks

```diff
@@ -228,7 +228,7 @@
 {% endmacro %}
 
 {% macro panel(content) %}
-  <article class="panel">
+  <article class="panel reading-width">
```

Comment:
I think it's better to use `div` for the macro. Particularly because it'll usually be embedded into an existing `article`. Better to change the semantics later than have wrong expectations.